doc/en/how-to/tmp_path.rst

```diff
@@ -109,6 +109,17 @@ rather than standard :class:`pathlib.Path` objects.
 .. note::
     These days, it is preferred to use ``tmp_path`` and ``tmp_path_factory``.
 
+    In order to help modernize old code bases, one can run pytest with the legacypath
+    plugin disabled:
+
+    .. code-block:: bash
+
+        pytest -p no:legacypath-tmpdir
```

Comment:
Shouldn't this be just `-p no:legacypath`?

Reply:
Good question

I used the registered name defined in `legacypath.py`.

From a quick test:

```python
def test_tmpdir(tmpdir):
      assert True
def test_tmpdir_factory(tmpdir_factory):
      assert True
```

either `no:legacypath` or `no:legacypath-tmpdir` triggers the expected error.

However, using `pytest --trace-config`, `legacypath` is listed there, so you are correct.

I will correct that.